<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>참여형 교육 솔루션</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../gui/Solution/logo.png</normaloff>../gui/Solution/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
border-image:url(:/images/bg1.jpg)0 0 0 0 stretch stretch;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
font-family:century gothic;
font-size: 24px;
}

#frame
{
	background:rgba(151, 214, 255,0.6);
	border-radius:15px;
}

#login {
	border-image:url(:/images/bg1.jpg)0 0 0 0 stretch stretch;
	background:rgba(0,0,0,0.3);
}

QLabel {
	color:#fff;
	background:transparent;
	font-size:30px;
}

QToolButton
{
	background:rgba(0, 0, 0,0);
	border-radius:40px;
}

#after_log_page {
border-image:url(:/images/bg1.jpg)0 0 0 0 stretch stretch;
}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="login">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1111</width>
        <height>751</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_2
{
	background:rgba(0,0,0,0.2);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>150</y>
         <width>441</width>
         <height>521</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit_id_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>345</width>
          <height>36</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>century gothic</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineEdit_id_2
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}</string>
        </property>
        <property name="placeholderText">
         <string>StudentID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_pw_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>310</y>
          <width>345</width>
          <height>36</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>century gothic</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineEdit_pw_2
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_login_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>380</y>
          <width>381</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>century gothic</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#Button_login_2
{
	background-color:rgba(151, 214, 255,0.6);
}</string>
        </property>
        <property name="text">
         <string>Log In</string>
        </property>
       </widget>
       <widget class="QToolButton" name="iconButton">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>-10</y>
          <width>291</width>
          <height>131</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_register">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>450</y>
          <width>381</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>century gothic</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#Button_register
{
	background-color:rgba(151, 214, 255,0.6);
}</string>
        </property>
        <property name="text">
         <string>Registration</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_username">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#label_username
{
	color:#fff;
	background:transparent;
	font-size:25px;
}</string>
        </property>
        <property name="text">
         <string>StudentID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_password">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>270</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#label_password
{
	color:#fff;
	background:transparent;
	font-size:25px;
}</string>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="after_log_page">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1111</width>
        <height>741</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_3
{
	background:rgba(0,0,0,0.2);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Button_makeRoom">
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>670</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Make Room</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_roomlist">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>1021</width>
         <height>441</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#tableWidget_roomlist
{
	background-color:rgba(255,255,255,0.7);
}</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1111</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame_7
{
	background-color: rgba(83, 175, 255, 0.7);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-10</y>
          <width>241</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_toHome">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>20</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>century gothic</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#Button_toHome
{
	background-color:rgba(151, 214, 255,0.6);
}
</string>
        </property>
        <property name="text">
         <string>LOG OUT</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="room_515">
     <property name="styleSheet">
      <string notr="true">#room_515
{
	border-image:url(:/images/bg1.jpg)0 0 0 0 stretch stretch;
}</string>
     </property>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1111</width>
        <height>731</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_4
{
	background:rgba(0,0,0,0.2);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1111</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame_8
{
	background-color: rgba(83, 175, 255, 0.7);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QToolButton" name="toolButton_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-10</y>
          <width>241</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>170</y>
         <width>591</width>
         <height>471</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="Chatting_log"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_msg"/>
          </item>
          <item>
           <widget class="QPushButton" name="send">
            <property name="text">
             <string>send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="room_516">
     <property name="styleSheet">
      <string notr="true">#room_516
{
	border-image:url(:/images/bg1.jpg)0 0 0 0 stretch stretch;
}</string>
     </property>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1111</width>
        <height>741</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_5
{
	background:rgba(0,0,0,0.2);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="chatting_log_516">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>100</y>
         <width>461</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_msg_516">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>620</y>
         <width>531</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="send_516">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>620</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="room_517">
     <property name="styleSheet">
      <string notr="true">#room_517
{
	border-image:url(:/images/bg1.jpg)0 0 0 0 stretch stretch;
}</string>
     </property>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1111</width>
        <height>741</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_6
{
	background:rgba(0,0,0,0.2);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="chatting_log_517">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>60</y>
         <width>461</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_msg_517">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>610</y>
         <width>531</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="send_517">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>610</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
